bugfixes.tex

```diff
@@ -25,21 +27,26 @@ \section{\Rsbdatazero Reads Order of Operations}
     \item Clear \Fsbbusy.
 \end{steps}
 
-\section{Hart reset behavior when \Fhaltreq is set}
+\subsection{Hart reset behavior when \Fhaltreq is set}
 
 When a hart comes out of reset and \Fhaltreq is set, the hart will immediately
 enter Debug Mode.
 
-\section{\Fmte only applies when action=0}
+\subsection{\Fmte only applies when action=0}
 
 The definition of \Fmte in Section 5.2.6 should state that \Fmte only affects
 triggers whose action is 0.
 
-\section{\Fsselect applies to \Fsvalue}
+\subsection{\Fsselect applies to \Fsvalue}
 
 In Section 5.2.13, when \Fsselect is 0 it ignores \Fsvalue.
 
-\section{Last trigger example}
+\subsection{Last trigger example}
 
 In the last example in Section B.9, the value for \Rtdatatwo 1 should be
 0xefff8675.
+
+\section{0.13.2}
+
+Fixed a formatting issue that caused step 1 in the Quick Access description
+from not being in the document.
```

Comment:
Fix grammar?  Maybe "...description to be missing from the rendered document"

Reply:
Done. Thanks.